### DISCUSSION TITLE
Specify automation_condition in dbt model's config block

### DISCUSSION THREAD

**Discussion by pin970** [dagster-io/dagster#24945] [Q&A]
Hello, is there a way to specify an `automation_condition` for a dbt model within the model's config block? Previously with auto-materialization I could do it like so:

```
{{
    config(
        meta={
            "dagster": {
                "auto_materialize_policy": {
                    "type": "eager"
                }
            }
        }
    )
}}
```

But if I change it to `automation_condition` it does not seem to work. Do you plan to add an option to configure `automation_condition` within the meta config block? I find it a bit inconvenient to use `DagsterDbtTranslator` when I only want to add simple automation for a single model.

**Comment by smackesey** [comment]
cc @OwenKephart

**Comment by OwenKephart** [comment]
Hi @pin970 -- as you note, the DagsterDbtTranslator is the surefire way of customizing this. However, this has come up before (quite recently) here: https://github.com/dagster-io/dagster/issues/25866, and I think it may be worth considering adding support at least for the `eager` condition to make migration easier here.

Are there other conditions you're considering specifying in this metadata? If so, then generally writing a bit of code in the translator to parse that metadata and add an automation condition based on what's there would probably make sense.
